docs/mobile/README.md

```diff
@@ -0,0 +1,28 @@
+# DataCamp for Mobile
+
+DataCamp for Mobile allows students to engage with their data science learning
+content anytime, anywhere (even [on the
+toilet](https://www.datacamp.com/community/blog/mobile-data-science)).
+
+A course is clearly subdivided into chapters, and lessons. A mobile course
```

Comment:
For courses, one of the most common problems is failing to understand the structure. For that reason, the documentation now has a section that explicitly mentions this. You might want to do the same. 

http://authoring.datacamp.com/courses/design/#terminology-and-structure

Reply:
Isn't that what this page does? I think the bullet formatting and the example makes it clearer though, will do.